src/main/java/seedu/modtracker/ModTracker.java

```diff
@@ -36,11 +45,24 @@ public String start() {
      *
      * @param name name of user
      */
-    public void runCommandLoopUntilExitCommand(String name) {
+    private void runCommandLoopUntilExitCommand(String name) {
         Parser parser = new Parser();
         while (!parser.isExit()) {
             String input = ui.readCommand();
-            parser.parse(input, modList, name, storage);
+            parser.parse(input, modList, name, storage, true);
+        }
+    }
+
+    private void loadData() {
+        try {
+            Parser parser = new Parser();
+            Scanner reader = storage.getReader();
+            while (reader.hasNext()) {
+                String input = reader.nextLine();
+                parser.parse(input, modList, "", storage, false);
+            }
+        } catch (FileNotFoundException e) {
+            ui.printErrorMessage(e.getMessage());
```

Comment:
Maybe can include a file creation here so the user need not manually create one

Reply:
The file is already created when the Storage object is initialised, this part is just a check when trying to read from the Storage object.